aim/aim_manager.py

```diff
@@ -203,7 +203,7 @@ def update(self, context, resource, fix_ownership=False,
                     if old_resource.user_equal(
                             old_resource_copy) and not force_update:
                         # Nothing to do here
-                        return old_resource
+                        return old_resource_copy
```

Comment:
nit: its probably the same thing but I feel that we should keep the existing code unchanged, i.e., just return the old_resource here like before.